recipes/minio-cpp/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class MinioCppConan(ConanFile):
+    name = "minio-cpp"
+    description = "MinIO C++ Client SDK for Amazon S3 Compatible Cloud Storage"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/minio/minio-cpp"
+    topics = ("minio", "s3", "storage")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8",
+            "clang": "7",
+            "apple-clang": "12",
+            "Visual Studio": "16",
+            "msvc": "192",
+        }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("curlpp/0.8.1.cci.20240530", transitive_headers=True)
+        self.requires("inih/58")
+        self.requires("nlohmann_json/3.11.3", transitive_headers=True)
+        self.requires("openssl/3.3.2")
+        self.requires("pugixml/1.14")
+        self.requires("zlib/1.3.1")
```

Comment:
Openssl and zlib should use version ranges!

Reply:
Thank you for your review.
This is the point I forgot to correct from my private recipe.
Fixed.